Rakefile

```diff
@@ -13,6 +12,24 @@ else
   task :default => :spec
 end
 
+task :appraisal do
+  for_all_gemfiles("exec", "rake")
+end
+
+namespace :appraisal do
+  task :install do
+    for_all_gemfiles("install")
+  end
+end
+
+def for_all_gemfiles(*args)
+  Dir.glob("gemfiles/rails_{5.2,6.0,edge}/Gemfile").each do |gemfile|
```

Comment:
Would be better to have just `gemfiles/*/Gemfile` no? So it does not need to change when we change compat matrix.

Reply:
Yes, that's what I wrote initially, but that was running `rails_edge` first (which fails), so I figure it would be better to enforce an order where stable versions come before unstable ones.

Reply:
I see. It does not run all tasks you mean? 

The order should not matter. If you want to evaluate all tasks before exiting with a status, then it should do something else like:
```ruby
gemfiles.map do |gemfile|
  sh(...) { |ok, status| next status }
end.all(&:success?)
```

Reply:
Maybe I can do something like

```ruby
 [*Dir.glob("gemfiles/rails_[5-9].[0-9]/Gemfile"), "gemfiles/rails_edge/Gemfile"]
```

That wouldn't require updating until Rails 10.

Reply:
I don't think we should impose naming conventions on folders.

I think there are two problems. One is that it exits on the first failure.

And the second is that it runs rails edge first. 
Wouldn't calling `.sort` sort the array? Or you could give it a block, right.

Reply:
Exiting on first failure is the current behavior of the current default task, so I preserved it here. If we want to change that for some reason, I think it should be discussed and done on a separate PR.

Regarding the order, sorting works and feels better than my approach! I'll update it.

Reply:
Updated.

Reply:
Cool 👍